.golangci.yml

```diff
@@ -88,4 +89,12 @@ linters-settings:
       - fmt.Fprintf
   gocritic:
     disabled-checks:
-      - commentFormatting
\ No newline at end of file
+      - commentFormatting
+  stylecheck:
+    checks:
+      # A function's error value should be its last return value.
+      # https://staticcheck.dev/docs/checks/#ST1008
+      - ST1008
+      # Importing the same package multiple times.
+      # https://staticcheck.dev/docs/checks/#ST1019
+      - ST1019
```

Comment:
ok, these will be removed in future PRs. Especially the second is concerning.

Reply:
why removed? we enable them here

Reply:
sorry I misread them to be in the "exclude" part. These are actually enabled, so we're good.